app/Models/Comment.php

```diff
@@ -34,4 +35,42 @@ public function article()
     {
         return $this->belongsTo('App\Models\Article');
     }
+
+    /**
+     * Retrieves the time elapsed since the comment was published from
+     * the current time.
+     * 
+     * @param string measurement years | months | weeks ...
+     * @param bool preformatted
+     * 
+     * @return mixed
+     */
+    public function time_elapse($measurement = '', $preformatted = true)
```

Comment:
Nice helper. 
Please use `camelCase` for methods, `snake_case` for variables, thanks.